Resources/views/Menu/sonata_menu.html.twig

```diff
@@ -33,6 +33,10 @@
             {%- endfor %}
         {%- endif %}
 
+        {%- if item.extra('route') is not empty and item.extra('route') == request.get('_route') %}
+            {%- set active = true %}
+        {%- endif %}
+
```

Comment:
> To simply test it and simplify view logic.
> Looking deeper in the code, I think the whole block should be in a MenuVoter instead.

I agree. Usually, I divide different task between different PR, so in this PR I try only add new option in menu. I think we can refactoring this block, but in another PR. @mvhirsch Thank you for you suggestion.